index.js

```diff
@@ -0,0 +1,47 @@
+const config = {
+  "env": {
+    "browser": true,
+    "es6": true,
+    "mocha": true,
+    "node": true
```

Comment:
thinking about more use cases, it could be split into several configs, using a main one as a parent, like https://github.com/segmentio/eslint-config

Im not sure if those flags were added considering those cases. If not, I can removed the unused ones.